IndexedDB/transaction-deactivation-timing.html

```diff
@@ -0,0 +1,143 @@
+<!DOCTYPE html>
+<meta charset=utf-8>
+<title>Transactions deactivation timing</title>
+<link rel="help" href="https://w3c.github.io/IndexedDB/#dom-idbdatabase-transaction">
+<script src=/resources/testharness.js></script>
+<script src=/resources/testharnessreport.js></script>
+<script src=support.js></script>
+<script>
+
+indexeddb_test(
+  (t, db, tx) => {
+    db.createObjectStore('store');
+  },
+  (t, db) => {
+    const tx = db.transaction('store');
+    const release_tx = keep_alive(tx, 'store');
+    assert_true(is_transaction_active(tx, 'store'),
+                'Transaction should be active after creation');
+
+    setTimeout(t.step_func(() => {
+      assert_false(is_transaction_active(tx, 'store'),
+                   'Transaction should be inactive in next task');
+      release_tx();
```

Comment:
Thanks. I ended up just moving it into the `keep_alive()` helper since it was getting copy/pastey